### PR TITLE
5.2.10 Compute resource password encryption

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: _includes/manuals/1.8/5.2.10_cr_encryption_key.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Compute resource passwords and secrets are stored on the Foreman database using a secret - the encryption key - and ciphered using AES-256-CBC.
+The encryption key can usually be found in config/initializers/encryption_key.rb. The value of the ENCRYPTION_KEY variable must be at least 32 bytes long.
```
Comment (domcleal) [inline _includes/manuals/1.8/5.2.10_cr_encryption_key.md]: On a package installation, this is a symlink to a file that's actually in /etc/foreman.  I wonder if it's worth saying that /etc is the location, and as a sidenote say it's symlinked to config.

File: _includes/manuals/1.8/5.2.10_cr_encryption_key.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+If you want to regenerate the key, you can run `foreman-rake security:generate_encryption_key`. Please remember that previously encrypted passwords will not be possible to decrypt with a different encryption key, so decrypt all passwords before changing your encryption key.
```
Comment (domcleal) [inline _includes/manuals/1.8/5.2.10_cr_encryption_key.md]: "previously encrypted passwords cannot be decrypted with.."

Comment (domcleal) [inline _includes/manuals/1.8/5.2.10_cr_encryption_key.md]: "previously encrypted passwords cannot be decrypted with.."

File: manuals/1.8/index.md [hunk 3]
```diff
@@ -205,6 +205,8 @@ version: 1.8
 {%include manuals/1.8/5.2.8_cr_rackspace.md %}
 ### 5.2.9 VMware Notes
 {%include manuals/1.8/5.2.9_cr_vmware.md %}
+### 5.2.10 Password encryption
```
Comment (domcleal) [inline manuals/1.8/index.md]: Nitpick, capitalise 'encryption' for consistency with other titles

Comment (domcleal) [inline manuals/1.8/index.md]: Nitpick, capitalise 'encryption' for consistency with other titles

**Comment by dLobatog** [comment]
@domcleal Updated, thanks for the comments!

**Comment by domcleal** [comment]
Thanks @eLobato!

**Pull request merged**